people_recognition_msgs/msg/People3D.msg

```diff
@@ -1 +1,4 @@
+sensor_msgs/Image image_rgb
```

Comment:
Is this just the same images that went in the node?

Reply:
Yes. This is something @reinzor asked me to add during the tournament to do a proper book keeping of the input-output pair.

Are there any requested changes here?